webpack.config.js

```diff
@@ -1,66 +1,48 @@
-const webpack = require("webpack");
-const path = require("path");
-const CleanWebpackPlugin = require("clean-webpack-plugin");
+const webpack = require('webpack');
+const path = require('path');
 
 const PACKAGE_ROOT_PATH = process.cwd();
 
 const config = {
-  entry: ["./src/index.js"],
+  entry: ['./src/index.ts'],
   output: {
-    path: path.resolve(PACKAGE_ROOT_PATH, "dist"),
-    publicPath: "dist/",
-    chunkFilename: "[name].js",
-    library: "ProtvistaUniprot",
-    filename: "protvista-uniprot.js"
+    path: path.resolve(PACKAGE_ROOT_PATH, 'dist'),
+    publicPath: '/',
+    chunkFilename: '[name].js',
+    library: 'ProtvistaUniprot',
+    filename: 'protvista-uniprot.js',
   },
-  target: "web",
-  devtool: "source-map",
+  target: 'web',
+  devtool: 'source-map',
   resolve: {
-    extensions: [".js"]
+    extensions: ['.ts', '.js'],
   },
   externals: {
-    d3: "d3",
-    litemol: "LiteMol"
+    d3: 'd3',
+    litemol: 'LiteMol',
   },
-  plugins: [new CleanWebpackPlugin([path.join(PACKAGE_ROOT_PATH, "dist")])],
```

Comment:
This has been removed, like when I updated uniprot-website and franklin for webpack 5. But were we supposed to remove it completely or was the plan to use `output.clean = true` there too since webpack 5 has it integrated?